apps/devhub/templates/devhub/email/new-key-email.ltxt

```diff
@@ -0,0 +1,7 @@
+The following API Key was added to your account:
```

Comment:
I'm pretty sure we want this to be localized. You can do so by wrapping it in `blocktrans`, something like:

````
{% load i18n %}{% blocktrans with key=key, manage_url=url('devhub.api_key')|absolutify %}
The following API Key was added to your account:
{{ key }}
...
{% endblocktrans %}
````
Example: https://github.com/mozilla/olympia/blob/master/apps/users/templates/users/email/author_added.ltxt